.devcontainer/devcontainer.json

```diff
@@ -1,9 +1,30 @@
 {
   "name": "helium-csharp",
+  // "image": "retaildevcrew/codespaces-dotnet",
```

Comment:
Actually, wouldn't it be better to use the image now rather than build from the dockerfile? the ci/cd is going to build and push the dockerfile so once that's done, new codespaces can be spun up using the image from the registry.

Reply:
Good point. I'll make the changes.